common/src/main/java/tachyon/worker/WorkerClient.java

```diff
@@ -299,10 +289,12 @@ public synchronized String lockBlock(long blockId) throws IOException {
    *
    * @throws IOException
    */
-  public synchronized void mustConnect() throws IOException {
+  @Override
+  public synchronized void connect() throws IOException {
```

Comment:
This  `connect` is so different from the `connect` method in superclass. Actually the `super` `connect` method is meant to be general enough for different implementations. Could you leave a TODO here and reminding us in followup PRs to merge the connect logic in this method into the super class.